lib/MySqlDatabaseManager.js

```diff
@@ -1,16 +1,185 @@
 var DatabaseManager = require('./DatabaseManager').default
-  , classUtils = require('./class-utils');
+  , classUtils = require('./class-utils')
+  , mysql      = require('mysql')
+  , Promise = require('bluebird')
+  , _ = require('lodash');
 
 /**
  * @constructor
  * @extends DatabaseManager
+ *
+ * Notes:
+ *   - Even though the method signature implicates that _masterConnectionUrl returns
+ *     an URL string, it actually returns an object because MySQL node lib
+ *     assumes that the database name is defined in the URL format.
+ *
  */
 function MySqlDatabaseManager() {
   DatabaseManager.apply(this, arguments);
+  this._masterClient = null;
+  this._cachedTableNames = null;
 }
 
 classUtils.inherits(MySqlDatabaseManager, DatabaseManager);
 
+/**
+ * @Override
+ */
+MySqlDatabaseManager.prototype.createDbOwnerIfNotExist = function() {
+  return this._masterQuery("CREATE USER IF NOT EXISTS ?@'localhost' IDENTIFIED BY ?", [this.config.knex.connection.user, this.config.knex.connection.password])
```

Comment:
I don't know much about mysql, so I have to ask... why there is hard coded 'localhost' when creating user?